crash_similarity.py

```diff
@@ -58,6 +58,15 @@ def read_corpus(fnames):
     return [gensim.models.doc2vec.TaggedDocument(trace, [i, signature]) for i, (trace, signature) in enumerate(elems)]
 
 
+def get_stack_traces_from_crashid(crash_id):
```

Comment:
Nit: the function should be called `get_stack_traces_from_crashid`, as it is returning a single stack trace 😃